iceberg/pom.xml

```diff
@@ -29,7 +29,7 @@
     <kryo-shaded.version>4.0.3</kryo-shaded.version>
     <iceberg.mockito-core.version>3.4.4</iceberg.mockito-core.version>
     <iceberg.avro.version>1.11.1</iceberg.avro.version>
-    <iceberg.kryo.version>5.2.0</iceberg.kryo.version>
+    <iceberg.kryo.version>5.5.0</iceberg.kryo.version>
```

Comment:
this may break hive - spark interoperability if spark is not on the same version of kryo. 
Can you please check once?